imsim/stamp.py

```diff
@@ -10,8 +12,32 @@
 from .psf_utils import get_fft_psf_maybe
 from .stamp_utils import get_stamp_size
 
+class ProcessingMode(Enum):
+    FFT = auto()
+    PHOT = auto()
+    FAINT = auto()
+
+
+@dataclass
+class StellarObject:
```

Comment:
Agreed. How about something like `ObjectCache`? I feel this is a pretty accurate description of what its purpose is, without making any assumptions of what type of object it might be.

Reply:
This is now called `ObjectCache` and reduced in size to only hold the index, flux and draw mode.